Policies/github-terms-of-service.md

```diff
@@ -98,25 +98,25 @@ You agree that you will not under any circumstances violate our [Acceptable Use
 
-This license does not grant GitHub the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
+This license does not grant GitHub the right to sell Your Content. It also does not grant GitHub the right to otherwise distribute or use Your Content outside of our provision of the Service, except that as part of the right to archive Your Content, GitHub may permit our partners to store and archive Your Content in public repositories in connection with the [GitHub Arctic Code Vault and GitHub Archive Program](https://archiveprogram.github.com/).
 
 #### 5. License Grant to Other Users
 Any User-Generated Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).
 
 If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content through the GitHub Service and to reproduce Your Content solely on GitHub as permitted through GitHub's functionality (for example, through forking). You may grant further rights if you [adopt a license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository). If you are uploading Content you did not create or own, you are responsible for ensuring that the Content you upload is licensed under terms that grant these permissions to other GitHub Users.
 
 #### 6. Contributions Under Repository License
-Whenever you make a contribution to a repository containing notice of a license, you license your contribution under the same terms, and you agree that you have the right to license your contribution under those terms. If you have a separate agreement to license your contributions under different terms, such as a contributor license agreement, that agreement will supersede.
+Whenever you add Content to a repository containing notice of a license, you license that Content under the same terms, and you agree that you have the right to license that Content under those terms. If you have a separate agreement to license that Content under different terms, such as a contributor license agreement, that agreement will supersede.
```

Comment:
What contributions have been ruled out by the change from "make a contribution" to "add content"? Would it cover IP (e.g. patentable algorithms) included in issue text?

Reply:
How about adding "Content" like comments/issues that are not explicitly mentioned in the license?

For example, the MIT License refers to software and associated documentation, which presumably does not include comments/issues/etc.

A legal opinion is probably needed here.

Reply:
Please see from https://github.blog/2020-10-15-updates-to-our-terms-of-service-and-our-privacy-statement/

> Clarification, for greater license certainty, that the inbound=outbound license model applies to issues and comments submitted to a repository when changes are made in response to or including those issues and comments

The goal of this is for clarity that feedback, discussion, and code review can be used under the project license. Just like a pull request against the repository, this may or may not cover patents or other rights, depending on the license.